--- nant-0.91/Makefile.0004~	2011-10-24 09:57:27.000000000 +0200
+++ nant-0.91/Makefile	2011-10-24 09:58:08.039489275 +0200
@@ -102,7 +102,7 @@ bootstrap/NAnt.DotNetTasks.dll:
 
 bootstrap/NAnt.CompressionTasks.dll:
 	$(MCS) $(DEBUG) -target:library -warn:0 -define:$(DEFINE) -out:bootstrap/NAnt.CompressionTasks.dll \
-		-r:./bootstrap/NAnt.Core.dll -r:bootstrap/lib/common/neutral/ICSharpCode.SharpZipLib.dll \
+		-r:./bootstrap/NAnt.Core.dll -r:ICSharpCode.SharpZipLib.dll \
 		-recurse:src${DIRSEP}NAnt.Compression${DIRSEP}*.cs src${DIRSEP}CommonAssemblyInfo.cs
 
 bootstrap/NAnt.Win32Tasks.dll:
--- nant-0.91/src/NAnt.Compression/NAnt.Compression.build.0004~	2010-02-17 06:08:02.000000000 +0100
+++ nant-0.91/src/NAnt.Compression/NAnt.Compression.build	2011-10-24 09:57:27.787511010 +0200
@@ -17,7 +17,7 @@
             </sources>
             <references basedir="${build.dir}/bin">
                 <include name="NAnt.Core.dll" />
-                <include name="${nant::scan-probing-paths(build.dir + '/bin','ICSharpCode.SharpZipLib.dll')}" />
+                <include name="ICSharpCode.SharpZipLib.dll" />
             </references>
             <resources>
                 <include name="*.resx"/>
--- nant-0.91/src/NAnt.Compression/Tasks/TarTask.cs.0004~	2011-09-17 22:08:22.000000000 +0200
+++ nant-0.91/src/NAnt.Compression/Tasks/TarTask.cs	2011-10-24 09:57:27.787511010 +0200
@@ -147,7 +147,7 @@ namespace NAnt.Compression.Tasks {
 
                 // do not use convert line endings of text files to \n, as this
                 // converts all content to ASCII
-                archive.AsciiTranslate = false;
+                archive.SetAsciiTranslation(false);
 
                 // process all filesets
                 foreach (TarFileSet fileset in TarFileSets) {
@@ -255,7 +255,7 @@ namespace NAnt.Compression.Tasks {
                 }
 
                 // close the tar archive
-                archive.Close();
+                archive.CloseArchive();
             } catch (Exception ex) {
                 // close the tar output stream
                 if (outstream != null) {
@@ -264,7 +264,7 @@ namespace NAnt.Compression.Tasks {
 
                 // close the tar archive
                 if (archive != null) {
-                    archive.Close();
+                    archive.CloseArchive();
                 }
 
                 // delete the (possibly corrupt) tar file
